docs/Documentation/Applications/comsol.md

```diff
@@ -15,18 +15,26 @@ Extensive documentation is available in the menu: **Help > Documentation**. For
 For instructional videos, see the [COMSOL website](https://www.comsol.com) Video Gallery.
 
 ## Building Models in the COMSOL GUI
-Before beginning, it is good practice to check the license status with:
+Before beginning, it is good practice to check the license status. To do so create a bash script named lmstat.comsol in your working directory and excute it:
 
+???+ example "License Checker Script"
+
+     ```bash
+     #!/bin/bash
+     COMSOL_LIC_DIR=/nopt/nrel/apps/software/comsol/6.1/comsol61/multiphysics/license/glnxa64
+     cd $COMSOL_LIC_DIR
+     ./lmstat -a --no-user-info -c ../license.dat
+     ```
+     
 ```
-[user@el3 ~]$ lmstat.comsol
+[user@kl3 ~]$ ./lmstat.comsol
 ```
 
-When licenses are available, COMSOL can be used by starting the COMSOL GUI which allows you to build models, run the COMSOL computational engine, and analyze results. The COMSOL GUI can be accessed through a [FastX desktop](https://eagle-dav.hpc.nrel.gov/session/) by opening a terminal and running the following commands:
+When licenses are available, COMSOL can be used by starting the COMSOL GUI which allows you to build models, run the COMSOL computational engine, and analyze results. The COMSOL GUI can be accessed through a [FastX desktop](https://kestrel-dav.hpc.nrel.gov/auth/ssh/) by opening a terminal and running the following commands:
 
 ```
-[user@ed3 ~]$ module purge
-[user@ed3 ~]$ module load comsol/6.0
-[user@ed3 ~]$ vglrun comsol
+[user@kl3 ~]$ module load comsol/6.1
+[user@kl3 ~]$ vglrun comsol &
 ```
 
 Because FastX desktop sessions are supported from DAV nodes shared between multiple HPC users, limits are placed on how much memory and compute resources can be consumed by a single user/job. For this reason, it is recommended that the GUI be primarily used to define the problem and run small-scale tests to validate its operation before moving the model to a compute node for larger-scale runs. For jobs that require both large-scale compute resources and GUI interactivity simultaneously, there is partial support for running the GUI from an X-enabled shell (ssh -Y ...) on a compute node by replacing the `vglrun comosl` command with:
```

Comment:
The prompt below should be changed to have a kestrel node name. 

Also, it might help if all of the steps for running the GUI from a compute node are shown (like including the command for requesting the interactive node), there have been some questions on that from users.